tests/test_tui_focused_filter_guard.py

```
@@ -106,3 +107,42 @@ def test_tui_status_names_remain_stable() -> None:
+    expression = str(value).strip()
+    if expression.startswith("${{") and expression.endswith("}}"):
+        expression = expression[3:-2].strip()
+    if expression == "true":
+        return True
+    if expression == "false":
+        return False
+    if expression == "inputs.mode == 'full'":
+        return mode == "full"
+    if variables is not None and expression in variables:
+        return variables[expression]
+    raise AssertionError(f"unsupported mode expression: {value!r}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)